submissions/andrewklmn/task_js_dom/app.js

```diff
@@ -0,0 +1,209 @@
+    },
+    {
+      menu: 'Reluctant Pornstar',
+      header: "The Reluctant Porn Star",
+      image: "img/adult.jpg",
+      description: "If the ingénue took a wrong turn on her road to fame, this is where she ended up. She’s not very talented, but she’s pretty — or she was before she got those ridiculous implants and started applying her makeup with a spackling knife.This is the third one.",
+      habitat: "Encino, Reseda, Northridge",
+      method: "How to attract her: Do you really want to? Well, I guess you can tell her she has the talent to be a mainstream actress. That would probably work. If it doesn’t, offer her some cocaine.",
+    },
+    {
+      menu: 'Hipster Chick',
+      header: "The Hipster Chick",
+      image: "img/hipster.png",
+      description: "She may have wanted to be an actor at some point, but she’s given that up. Now, she’s too cool for school and incapable of saying or doing anything that’s not dripping with irony. She loves coffee and cigarettes, doesn’t own a TV and only listens to music on vinyl.",
+      habitat: "Silverlake, Echo Park",
+      method: "How to attract her: Wear horn-rimmed glasses and a fedora, and don’t give a shit about anything.",
+    },
+    {
+      menu: 'Flower Child',
+      header: "The Flower Child",
+      image: "img/flower.jpg",
+      description: "Most of these specimens have migrated north to San Francisco or Portland, but they can still be seen sometimes in the Los Angeles area if you know where to look. You can recognize her from her sundress and unshaved armpits. She believes in free love, which is good, but she’s probably a raw vegan, so I hope you’re not hungry.",
+      habitat: "Venice, Santa Monica",
+      method: "How to attract her: Don’t shave, use deodorant or have a job, and always carry around a little weed (Don’t worry, it’s practically legal here).",
+    },
+    {
+      menu: 'Gold Digger',
+      header: "The Gold Digger",
+      image: "img/digger.jpg",
+      description: "Sadly, this is one of the more common types of L.A. women, but luckily they’re easy to spot. The twist here is that you don’t necessarily have to have a lot of money, you just have to have the potential to make it. So if you’re a junior agent or work in the mailroom at an entertainment law firm, she’ll invest her time in you. But if you don’t come through, buddy, you are screwed. All she wants is the house on the hill and the fancy car, so if you’re just looking for a trophy wife, then take your pick.",
+      habitat: "Beverly Hills, Hollywood Hills, any Hills, really.",
+      method: "How to attract her: What are you, an idiot? Just flash your cash!",
+    },
+  ],
+  activeIndex: null,
+};
+
+const menuToogleClickListener = (event)=>{
+  let elem = event.target;
+  let aside = document.querySelector('aside');
```

Comment:
Let's use here `const` instead of `let` for these variables. See the difference [here](https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/).
Also, I'd recommend you to move these variables to the beginning of this file where you declared your constants.
Because now you querying your DOM and create a new variable on each click of the button. It can reduce the performance of your app